spec/info.yaml

```
@@ -8,4 +8,4 @@ contact:
 license:
   name: Apache 2.0
   url: http://www.apache.org/licenses/LICENSE-2.0.html
-version: 3.7.1
+version: 3.8.1
```

I think this should be version 3.8.0
```suggestion
version: 3.8.0
```

---

crap